samples/net/udp/sample.yaml

```diff
@@ -0,0 +1,19 @@
+sample:
+  name: UDP sample
+tests:
+  sample.net.udp:
+    build_only: true
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+      - nrf9161dk_nrf9161_ns
+      - thingy91_nrf9160_ns
+      - nrf7002dk_nrf5340_cpuapp
+    platform_allow:
+      - nrf9160dk_nrf9160_ns
+      - nrf9161dk_nrf9161_ns
+      - thingy91_nrf9160_ns
+      - nrf7002dk_nrf5340_cpuapp
+    tags: ci_build
+    extra_configs:
+      - CONFIG_WIFI_CREDENTIALS_STATIC_SSID="ssid"
```

Comment:
These configs should only be used for a configuration that is built for wifi. So I think its better to create separate configurations for LTE and WIFI

Reply:
Yeah agree, I was hoping to avoid that :smile: But ill make the change